### ISSUE TITLE
Newsletter Captcha: Sizing + Spacing

### ISSUE THREAD

**Issue by julefairfood** [shopware/shopware#11835]
### Shopware Version

cloud

### Affected area / extension

Platform(Default)

### Actual behaviour

Since June, European law has required shop pages to be accessible for screen readers. However, according to relevant accessibility tools, there are some issues with the newsletter captcha. Both the button and the link to "FriendlyCaptcha," which serve as a linked user interface, have been criticized in various reports as being too small or lacking in spacing. Adequate font size is very important within the context of the BSFG.

<img width="583" height="589" alt="Image" src="https://github.com/user-attachments/assets/4036a8b1-8623-44d1-888e-6697c7a7d45a" />

### Expected behaviour

Larger font and more spacing in the button. I know this is possible with manual CSS adjustments. However, we use Shopware SaaS, lack development expertise, and therefore want a clean solution.

### How to reproduce

for example by conducting a Lighthouse report

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by MalteJanz** [comment]
@BrocksiNet I guess this topic fits more for the Storefront team? The captcha topic is not part of our domain

**Comment by fabianhueske** [comment]
Yes. @tobiasberge already knows about this! 😊

**Comment by joberthel** [comment]
Thank you for raising this issue and for providing the screenshot!

You are right: the Friendly Captcha shown in your screenshot has the accessibility issue you described. It does not comply with [WCAG 2.2 AA – 2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum)

However, this should not currently be a blocker from a legal compliance perspective, as the current EN 301 549 standard still references WCAG 2.1. The Target Size (Minimum) criterion was only introduced at Level AA with WCAG 2.2.

I noticed, though, that you are using the old legacy version (v1) of Friendly Captcha. I would recommend switching to the newer v2 version. In my testing, v2 does not show this issue and appears to be fully WCAG 2.2 AA compliant.

You can select the Friendly Captcha version in the Administration under **Settings > Basic information > CAPTCHA**.

<img width="990" height="476" alt="Image" src="https://github.com/user-attachments/assets/b3b71a72-3154-4528-9a30-c70efa78fea5" />

I am closing this issue for now. Thanks again for bringing this to our attention!
